### ISSUE TITLE
VSCode incorrectly detects rebase in progress

### ISSUE THREAD

**Issue by wenfangdu** [microsoft/vscode#109350]
- VSCode Version: v1.50.1
- OS Version: Windows 10

Steps to Reproduce:
1. Use command line to do interactive rebase
2. After rebase completed, the message box was still outlined

![image](https://user-images.githubusercontent.com/28700378/97142176-1f740900-179b-11eb-8c42-fd5583ec8532.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by joaomoreno** [comment]
Do you have any of the following files?

- `.git/REBASE_HEAD`
- `.git/rebase-apply`
- `.git/rebase-merge`

?

**Comment by wenfangdu** [comment]
@joaomoreno Ah, I just found it's not reproducible every time. When the files you mentioned are gone, this behavior wouldn't happen, might be other reasons such as I'm running `yarn serve` in the background, it might somehow interfered the file system and caused this. I'll keep watching to see if that's the case. 

Also, when this happened, I tried to click refresh button in the scm panel, in the command line, `git status` showed the working directory was clean, but VSCode falsely considered it was still rebasing. Could this be adjusted so that refresh button would reset the outline if rebase is done not through detecting the files you mentioned but the actual `git status`?
